flink-python/pyflink/version.py

```diff
@@ -20,4 +20,4 @@
 The pyflink version will be consistent with the flink version and follow the PEP440.
 .. seealso:: https://www.python.org/dev/peps/pep-0440
 """
-__version__ = "1.13.0"
+__version__ = "1.13.dev0"
```

Comment:
@HuangXingBo If we do a release, where is this being changed now?

Reply:
It will be changed in `create_release_branch.sh`